tailwind.config.ts

```diff
@@ -14,11 +14,12 @@ const config: Config = {
         'color-1': '#0f3b26', // Main Green
         'color-2': '#f9f7f7', // NavBar Off White
         'color-3': '#575555', // Aside Gray
+        'dark-gray': '#222', // Dark Gray
```

Comment:
can you use `color-4` instead, I know its less descriptive but named colors is bad for theme management, colors should be irrespective on content and more about location of use, This entire color pallete needs to be rewritten tbh to use a good pallete with names like `primary-bg` and stuff.

Reply:
fixed